pkg/controller/install/install_controller.go

```diff
@@ -176,37 +154,78 @@ func (r *ReconcileInstall) Reconcile(request reconcile.Request) (reconcile.Resul
 		return reconcile.Result{}, err
 	}
 
+	updateDomain(r.client, reqLogger)
+	updateServiceNetwork(r.client, reqLogger)
+
+	return reconcile.Result{}, nil
+}
+
+func getServiceNetwork(client client.Client, logger logr.Logger) string {
+
+	networkConfig := &configv1.Network{}
+
+	serviceNetwork := ""
+	if err := client.Get(context.TODO(), types.NamespacedName{Name: clusterObjectName}, networkConfig); err != nil {
+		logger.Info("Network Config is not available.")
+	} else if len(networkConfig.Spec.ServiceNetwork) > 0 {
+		serviceNetwork = strings.Join(networkConfig.Spec.ServiceNetwork, ",")
+		logger.Info("Network Config is available", "Service Network", serviceNetwork)
+	}
+
+	return serviceNetwork
+}
+
+func getDomain(client client.Client, logger logr.Logger) string {
+	ingressConfig := &configv1.Ingress{}
+	domain := ""
+	if err := client.Get(context.TODO(), types.NamespacedName{Name: clusterObjectName}, ingressConfig); err != nil {
+		logger.Info("Ingress Config is not available.")
+	} else {
+		domain = ingressConfig.Spec.Domain
+		logger.Info("Ingress Config is available", "Domain", domain)
+	}
+
+	return domain
+}
+
+func updateDomain(client client.Client, logger logr.Logger) {
```

Comment:
I think it makes sense to follow the pattern of returning an `error` from these functions, so that the request gets re-queued in the event of some external error, e.g. network blip. See how the `install/deleteObsoleteResources/checkForMinikube` fns are invoked in `Reconcile` for the expected pattern.

Reply:
fyi, I just committed these in the interim before I merged in your changes.   I didn't even see your pattern before I committed